### ISSUE TITLE
Jaws 2018 doesn't read anything on fields configured as role="search"

### ISSUE THREAD

**Issue by manojirala** [FreedomScientific/VFO-standards-support#94]
Steps to Reproduce:
1. Configure a field with role="Search" 
2. On IE 11, turn on Jaws 2018 and keep tabbing until focus goes to search region(search field). Observe that Jaws doesn't read anything. With Jaws 18 and earlier versions, Jaws says "Region, Search Region). But Jaws 2018 doesn't say anything. Jaws 2018 does honor role="main". It says "Main Region" which is expected.
Jaws 2018 does read "Region Search text edit" when the focus is within the text input field. That is not the problem we are facing. The problem is with the initial focus where the entire Search area is highlighted. Please check on the screenshot attached. I'm also attaching a code sample. Please save the text file as .html, open on IE11 and test it with Jaws 2018.
![rolesearch](https://user-images.githubusercontent.com/40472160/41737326-94f80bae-7554-11e8-821d-6922b465e5fe.png)

[RoleSearchSample.txt](https://github.com/FreedomScientific/VFO-standards-support/files/2124815/RoleSearchSample.txt)

RoleSearch

**Comment by jducrot** [comment]
This is not happening with JAWS 20 on IE11. The region is properly announced along with the input and the button. Here is a codepen for easier review: https://cdpn.io/torcud/debug/oNjpjPa/gaMeYzeVPqLM
